plugins/frontend/backstage-plugin-github-pull-requests/src/components/GithubPullRequestsContext.tsx

```diff
@@ -0,0 +1,23 @@
+import React, { FC, useState, useContext } from "react"
+import { PrState, PullRequestsContextData } from "../types"
+
+const GithubPullRequestsContext = React.createContext<PullRequestsContextData>(
+    {
+        prState: { open: { etag: "", data: [] }, closed: { etag: "", data: [] }, all: { etag: "", data: [] } },
```

Comment:
Maybe create an `initialValue` variable that can be cloned for use in here and the useState hook call below. This is getting a bit complex so easier to keep a single item up to date that way.

Reply:
Yeah, done